db/internal_stats.cc

```diff
@@ -776,6 +784,27 @@ bool InternalStats::HandleIsWriteStopped(uint64_t* value, DBImpl* db,
   return true;
 }
 
+bool InternalStats::HandleEstimatedOldestKeyTime(uint64_t* value,
+                                                        DBImpl* /*db*/,
+                                                        Version* /*version*/) {
+  // TODO(yiwu): The property is currently available is there's no compaction,
+  // e.g. FIFO compaction with compaction_options_fifo.allow_compactions
+  // disbled. This is because we don't propagate oldest_key_time on
+  // compaction.
```

Comment:
Can we add another condition in this condition to return false if compaction_style is not fifo or if fifo's allow_compaction is true? 
Apologize for requesting one more change 😞  .